fuzz/Cargo.toml

```diff
@@ -5,6 +5,9 @@ authors = ["Arvid Norberg <arvid@chia.net>"]
 publish = false
 edition = "2021"
 
+[workspace]
+members = ["."]
+
```

Comment:
this change seems redundant, and it's also not making the change suggested by the PR title